api/service.py

```
@@ -822,7 +823,7 @@ async def get_anomaly_history(
 
 
 @app.post("/api/v1/chaos/inject")
-async def inject_fault(request: ChaosRequest):
+async def inject_fault(request: ChaosRequest, api_key: APIKey = Depends(require_permission("admin"))):
```

Ruff (0.14.10)

Undefined name `ChaosRequest`

(F821)